content/posts/newsletter/2020-05-01.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Draft Title - Date
+save_as: 2020/06/newsletter-09-june/index.html
+url: 2020/06/newsletter-09-june/
+date: 2020-06-09 11:00
+author: emus
+blog_id: newsletter
+category: Newsletter
+---
+# XMPP@home - 9 June 2020
+## Intro
```

Comment:
Please remove this line as this pollutes, and brings nothing but loss of screen estate and information overload